docs/developer/eventing/sources/sinkbinding/getting-started.md

```diff
@@ -186,11 +186,10 @@ Create a `SinkBinding` object that directs events from your subject to the sink.
     Create a `SinkBinding` object by running:
 
     ```bash
-    kn source binding create <name> \
-      --namespace <namespace> \
-      --subject "<subject>" \
-      --sink <sink> \
-      --ce-override "<cloudevent-overrides>"
+    kn source binding create bind-heartbeat \
+     --subject "Job:batch/v1:app=heartbeat-cron" \
+     --sink event-display \
+     --ce-override "sink=bound"
```

Comment:
I think this command was valid. This command has placeholders for users to input their own values. Could you replace this command please?

The [issue](https://github.com/knative/docs/issues/4178#issuecomment-912752920) is referring to the example command on lines 211-215. If this example command is causing confusion, it might make sense to just delete it.

Reply:
done!.. i did not delete the 211-215 one tho.. should we? Im not positive.

Reply:
The command with the placeholders needs to be before the "Where" section on line 194 because that section has the description of what the user should replace the placeholder with.

Reply:
In terms of the example (`kn source binding create bind-heartbeat...`), I'm not sure whether it is confusing. 

The intention of this topic is for users to create their own sinkbinding with their own values, not just run the example commands. The reason why I suggested deleting the example is because I thought there might be some confusion about whether you are supposed to run the example command as well. 

If you don't think the example is confusing you can keep it, but it should go under the line with "For example".